dm/master/election.go

```diff
@@ -30,13 +30,19 @@ func (s *Server) electionNotify(ctx context.Context) {
 			// output the leader info.
 			if leader {
 				log.L().Info("current member become the leader", zap.String("current member", s.cfg.Name))
+				err := s.coordinator.Start(ctx, s.etcdClient)
```

Comment:
If before the old leader called `s.coordinator.Stop()`, the new leader called `s.coordinator.Start`, is it OK?

Reply:
I think we can add a validator before RPC to handle that check if the dm-master is a leader. that may be related to the gRPC proxy.

Reply:
OK